cw-orch-daemon/Cargo.toml

```diff
@@ -45,7 +45,7 @@ bech32 = { version = "0.11.0", default-features = false, features = ["alloc"] }
 hex = { version = "0.4.3" }
 ripemd = { version = "0.1.3" }
 tokio = { workspace = true, features = ["full"] }
-tonic = { workspace = true, features = ["tls", "tls-roots"] }
+tonic = { workspace = true, features = ["tls"] }
```

Comment:
Reallt removing the tls roots here ?

Reply:
It works without so not sure why we should enable it.